.github/workflows/maven.yml

```diff
@@ -22,8 +22,8 @@ jobs:
       matrix:
         # test against latest update of each major Java version, as well as specific updates of LTS versions:
         RUNTIME: [ol, wlp]
-        RUNTIME_VERSION: [25.0.0.6]
-        java: [21, 17, 11, 8]
+        RUNTIME_VERSION: [25.0.0.10]
```

Comment:
we generally like to test the LTS releases (3/6/9/12)...so can we test Java 8/11/17/21 with 25.0.0.9 and Java 25 with 25.0.0.10? Also, since we have so many versions, let's do some more exclusions like we did for 8/11 below. We can skip `wlp` for 8 and 17 and skip `ol` for 11 and 21. Make the change for both sections (linux and windows).